### ISSUE TITLE
№19

### ISSUE THREAD

**Issue by someguy123321** [thelink2012/novisualrep#6]
My english may be very bad so I'm sorry for it.
In underground 1, I have a trouble with magazine 19. No matter how many times I've beat the game, it hasn't been unlocked a single time. Any ideas why it happens to me? Game version 1.4. No cd from SPiRiTY. I don't have any other modifications except novisualrep and widescreen fix. According to readme 19 magazine unlocks after 112 race which is end of underground mode and... Maybe that's the problem? Or did everything work as it should during testing? After finale race, I see only one message about $, but not about any magazine cover. Please help.

**Comment by thelink2012** [comment]
Hey @someguy123321. You should be able to find the magazine by going into Underground Mode (after you finished it) and then Magazines. But yeah, I don't recall any pop-up after end game regarding it. Let me know if you can't find it.

**Comment by someguy123321** [comment]
I was checking "magazines", of course. №19 was locked every time. But the problem was solved by itself. If may say so. For each game walkthrough, I used my ancient save. With all rewards for style points. Vinyls, pre-tuned cars, etc. That save VERY old. It's been created around since I was study in school or something. Mod can't work properly with it for some reason. When I created a new save file, all magazines were unblocked according to what is written in readme. So everything is working properly for me now.
